tests/data_result/module_080_130/demo/demo_product_category.xml

```diff
@@ -1,5 +1,9 @@
 <odoo>
+
+    <data>
         <record name="demo_category" model="product.category">
             <field name="name">My Demo Category</field>
         </record>
+    </data>
```

Comment:
Can the solution don't add ``<data>`` until we have ``<data noupdate="1">`` in the source?

Reply:
Maybe better in another PR. This one was not thought out all that well. But it was fixing an important bug